week-1-ts-ecosystem/w01e04-light-me-up/src/ErrorHandler.ts

```diff
@@ -0,0 +1,5 @@
+export class ErrorHandler {
+  print(errorMsg: string) {
+    document.body.innerHTML = errorMsg;
+  }
+}
```

Comment:
`ErrorHandler` nie musi tutaj być klasą. Chyba że @kdrzymalski chciałeś przećwiczyć temat `IoC` (Inversion of Control) i dodatkowo dodać jeszcze jedną zależność do `LightBulb` w ten sam sposób - jako instancję.

Reply:
Bardziej miałem na myśli jakąś dobrą praktykę żeby zależności wstrzykiwać przez konstruktor/metodę i wtedy łatwiej testować daną klasę